todo_app/app.py

```diff
@@ -1,28 +1,61 @@
+from board_item import Board_item
 from flask import Flask, render_template, redirect, url_for, request
 
 from todo_app.flask_config import Config
 from todo_app.data import session_items as session
 
+import requests
+import os
+from dotenv import load_dotenv
+import json
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
+load_dotenv()
+host =  os.getenv('HOST')
+api_key = os.getenv('API_KEY')
+api_secret = os.getenv('API_TOKEN')
+board_id = os.getenv('BOARD_ID')
+todo_board_list_id = os.getenv('TODO_LIST_ID')
+done_list_id = os.getenv('DONE_LIST_ID')
```

Comment:
It doesn't look like you're using this done list anywhere, instead you're determining the cards status by the dueComplete flag. If you want to stick with that (it does limit you to 2 statuses, but isn't a huge issue) then we should clear out this unused variable